src/utils/transaction-guards.ts

```diff
@@ -47,32 +54,15 @@ export const isOwner = (safeOwners: AddressEx[] | NamedAddress[] = [], walletAdd
   return safeOwners.some((owner) => sameAddress(owner.address, walletAddress))
 }
 
-export const isMultisigExecutionDetails = (value?: DetailedExecutionInfo): value is MultisigExecutionDetails => {
-  return value?.type === 'MULTISIG'
+// Narrows `TransactionDetails` -> when requesting a single transaction
```

Comment:
The guards are for `TransactionDetails['detailedExecutionInfo']`.

Reply:
maybe the comments are unnecessary after all. I thought they would help clarifying the situations where the code deals with `ExecutionInfo` vs `DetailedExecutionInfo`